Runtime/Internal/JsRuntime.jspre

```diff
@@ -61,10 +61,11 @@ class JsSpecialReference {
     }
     GetSharedArray() {
         var data = this.data;
-        var sharedArray = this.runtime.arrayBuilder(data.pointer, data.typeCode, data.length);
+        var sharedArray = this.runtime.arrayBuilder(data.typeCode, data.pointer, data.length);
```

Comment:
This was an propagated change from a year ago. Can you make sure that it's the right parameters to this function?

Reply:
@MHDante Do I just make an array and see if it works? What's the best way to test this?